packages/common/src/gckms/GckmsConfig.js

```diff
@@ -79,32 +65,48 @@ function getDefaultStaticConfig() {
   return defaultConfig;
 }
 
-function getNetworkName() {
-  if (isPublicNetwork(argv.network || "")) {
-    // Take everything before the underscore:
-    // mainnet_gckms -> mainnet.
-    return argv.network.split("_")[0];
+function getGckmsConfig(keys = arrayify(argv.keys), network = argv.network) {
+  let configOverride = {};
+
+  // If there is no env variable providing the config, attempt to pull it from a file.
+  // TODO: this is kinda hacky. We should refactor this to only take in the config using one method.
+  if (process.env.GCKMS_CONFIG) {
+    // If the env variable is present, just take that json.
+    configOverride = JSON.parse(process.env.GCKMS_CONFIG);
+  } else {
+    // Import the .GckmsOverride.js file if it exists.
```

Comment:
this is backward compatible to still work with how we do things on truffle, right? clearly does not matter as we plan on removing Truffle in full but in the interim will things we used before still function?

Reply:
I have tested it with `truffle console` and it seems to still work!

Reply:
The vast majority of this code remains unchanged -- it was just moved to a function.

Reply:
yup. just wanted to double check :)